package.json

```diff
@@ -17,12 +17,10 @@
     "build:storybook": "build-storybook",
     "prebuild:storybook": "npm run build:design-tokens && npm run build:svg",
     "prebuild:design-tokens": "npm run clean:design-tokens",
-    "prebuild:svg": "npm run clean:svg",
     "build:design-tokens": "node ./packages/design-tokens/scripts/build.js",
     "build:svg": "node ./packages/icons/scripts/build.js",
     "clean": "rimraf dist",
     "clean:design-tokens": "rm -rf ./dist/packages/design-tokens && rm -rf ./packages/design-tokens/dist/",
-    "clean:svg": "rm -rf ./dist/packages/icons && rm -rf ./packages/icons/dist/",
```

Comment:
Removing this "clean" step because when we merge to master, Jenkins runs the Typescript compiler first, and then runs the icon build again (including the "clean" step) before building and deploying Storybook.

This resulted in the JavaScript files that were generated in `dist/icons/` to be removed right before we publish to NPM.

Check this log to see what I'm talking about: https://jenkins.mesosphere.com/service/jenkins/job/frontend/job/UI-Kit%20Pipeline/job/master/208/console